tmcprototype/SdpSubarrayLeafNode/test/SdpSubarrayLeafNode_test.py

```diff
@@ -137,7 +142,7 @@ def test_Configure(self, tango_context, create_sdpsubarray_proxy):
                      ':{"scanParameters":{"12346":{"fieldId":0,"intervalMs":2800}}}}}'
         tango_context.device.Configure(test_input)
         time.sleep(1)
-        assert CONST.STR_CONFIGURE_SUCCESS in tango_context.device.activityMessage
+        assert create_sdpsubarray_proxy.obsState in [CONST.ENUM_CONFIGURING,CONST.ENUM_READY]
```

Comment:
Should we only assert with READY obsState?

Reply:
Yes.